src/System.Collections.Concurrent/ref/System.Collections.Concurrent.cs

```diff
@@ -64,6 +64,9 @@ public partial class ConcurrentBag<T> : System.Collections.Concurrent.IProducerC
         bool System.Collections.ICollection.IsSynchronized { get { throw null; } }
         object System.Collections.ICollection.SyncRoot { get { throw null; } }
         public void Add(T item) { }
+#if netcoreapp11
```

Comment:
The default build configuration is netcoreapp11 and so you don't actually need to do these #ifdef's any longer.